specs/core/0_beacon-chain.md

```diff
@@ -360,6 +372,13 @@ class BeaconBlockHeader(Container):
     parent_root: Root
     state_root: Root
     body_root: Root
+```
+
+#### `SignedBeaconBlockHeader`
+
+```python
+class SignedBeaconBlockHeader(Container):
+    msg: BeaconBlockHeader
```

Comment:
`msg` => `message` (the spec doesn't otherwise use abbreviations)

Reply:
On second thought, maybe we want `signed_header.header`.

Reply:
I like `msg` because 9/10 times the field is used for `bls_verify(pubkey, msg, signature, domain)` and you have a standard name that is explicitly about the inner container, not confusing with the signed header. `message` works too and is more consistent indeed.

Reply:
> `message` works too and is more consistent indeed

Please make it `message` :)

Reply:
Done